src/sanitizers.rs

```diff
@@ -238,4 +281,32 @@ mod tests {
         let result = strip_stuff_between_messages(input);
         assert_eq!(result, expected);
     }
+
+    #[test]
+    fn excess_tag86_are_stripped() {
+        let input = "\
+            :20:vvvvv\r\n\
+            :86:hello\r\n\
+            multi line string\r\n\
+            here is ok\r\n\
```

Comment:
Let's add a curve ball here:
```suggestion
            here is ok even with date that looks like a tag 20:10:43\r\n\
```